app/controllers/posts_controller.rb

```diff
@@ -2,7 +2,7 @@
 class PostsController < ApplicationController
   def index
     # byebug
-    @posts = Post.all
+    @posts = Post.order('created_at DESC').page(params[:page]).per(1)
   end
```

Comment:
Always try to avoid ordering by created_at, we can always achieve the same by ordering with ID